rust/processor/src/db/common/models/emojicoin_models/event_utils.rs

```diff
@@ -98,7 +101,13 @@ impl EventGroupBuilder {
     }
 
     pub fn add_periodic_state(&mut self, periodic_state_event: PeriodicStateEvent) {
-        debug_assert!(self.periodic_state_events.len() < 7);
+        if self.periodic_state_events.len() >= 7 {
+            let msg = "EventGroups can only have up to seven PeriodicStateEvents.";
+            panic!(
+                "{} Existing {:?}, attemped to add: {:?}",
```

Comment:
typo

Reply:
Fixed in 8e6a2a1f9cbb6f0a792b10037b23d5a65bf8b343